pkgs/data/themes/lightly-qt/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  mkDerivation,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  extra-cmake-modules,
+  kdecoration,
+  plasma-workspace,
+  qtbase,
+  qt5
+}:
+
+mkDerivation rec {
+  pname = "lightly-qt";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "Luwx";
+    repo = "Lightly";
+    rev = "v${version}";
+    sha256 = "0qkjzgjplgwczhk6959iah4ilvazpprv7yb809jy75kkp1jw8mwk";
+  };
+
+  buildInputs = [
+    kdecoration
+    plasma-workspace
+    qtbase
+    qt5.qtx11extras
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  meta = {
+    description = "A fork of breeze theme style that aims to be visually modern and minimalistic";
+    homepage = "https://github.com/Luwx/Lightly";
+    license = lib.licenses.gpl2;
```

Comment:
`licenses.gpl2` shouldn't be used anymore; from the license in the repo it looks like `licenses.gpl2Plus` would be a better fit. (the `COPYING` file is just the license text of GPLv2, and the source files allow "any later version".)

Reply:
Thanks. Fixed.